neard/src/cli.rs

```diff
@@ -493,12 +493,29 @@ pub(super) struct RecompressStorageSubCommand {
     /// Directory where to save new storage.
     #[clap(long)]
     output_dir: PathBuf,
+
+    /// Keep data in ColPartialChunks column.  This is always true if node is
+    /// not an archival node.
+    #[clap(long)]
+    keep_partial_chunks: bool,
```

Comment:
What is the default value here? false?

so if someone runs the recompress after we submit this PR (assuming they have archival node) - they will automatically 'lose' those columns, right?

Maybe we should wait with this PR until we finish the code for computing partial_chunks/trie changes on the fly?

Reply:
> What is the default value here? false?

Yep.

> so if someone runs the recompress after we submit this PR (assuming they have archival node) - they will automatically 'lose' those columns, right?

Yep.

> Maybe we should wait with this PR until we finish the code for computing partial_chunks/trie changes on the fly?

The code to recompute partial chunks is already merged in #6377. For recomputing trie changes we decided we won’t bother at least for now. The column only matters when non-archival node is doing GC.

There is #6439 still pending but I think it doesn’t matter that much which one of those gets merged first.